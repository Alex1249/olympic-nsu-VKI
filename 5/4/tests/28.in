10
e v f s n c z u i o
