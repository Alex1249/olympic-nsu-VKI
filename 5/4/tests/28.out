e v f s n c z u o i
